### PR TITLE
Added interactive tests for login and list-clusters commands

### PR DESCRIPTION
**Pull request by yuce**

### CODE REVIEW

**Review by kutluhanmetin — DISMISSED** [review]
Just two minor naming suggestions, LGTM!

File: base/commands/viridian/viridian_it_test.go [hunk 1]
```diff
@@ -37,8 +37,10 @@ func TestViridian(t *testing.T) {
 		f    func(t *testing.T)
 	}{
 		{"listClusters", listClustersTest},
+		{"istClustersInteractive", listClustersInteractiveTest},
```
[the remark this answers is not in the document]
Comment (yuce) [reply base/commands/viridian/viridian_it_test.go]: Updated at: https://github.com/hazelcast/hazelcast-commandline-client/pull/230/commits/f483eeebf937736ec2ce489834e47426d8c54967

File: base/commands/viridian/viridian_it_test.go [hunk 2]
```diff
@@ -37,8 +37,10 @@ func TestViridian(t *testing.T) {
...
 		{"loginWithEnvVariables", loginWithEnvVariablesTest},
 		{"loginWithParams", loginWithParamsTest},
+		{"loginWithParamsInteractive", loginWithParamsInteractiveTest},
```
[the remark this answers is not in the document]
Comment (yuce) [reply base/commands/viridian/viridian_it_test.go]: Updated at: https://github.com/hazelcast/hazelcast-commandline-client/pull/230/commits/f483eeebf937736ec2ce489834e47426d8c54967

[the remark this answers is not in the document]
Comment (yuce) [reply base/commands/viridian/viridian_it_test.go]: Updated at: https://github.com/hazelcast/hazelcast-commandline-client/pull/230/commits/f483eeebf937736ec2ce489834e47426d8c54967

**Pull request merged**